batch/batch/server/server.py

```diff
@@ -7,12 +7,14 @@
 from collections import Counter
 import logging
 import threading
-from flask import Flask, request, jsonify, abort, render_template
 import kubernetes as kube
 import cerberus
 import requests
+import aiohttp_jinja2
+import jinja2
+from aiohttp import web
```

Comment:
Could we use uvloop? Should be faster than the default event loop:

```
import asyncio
import uvloop
asyncio.set_event_loop_policy(uvloop.EventLoopPolicy())
```

Reply:
I support this.